### PR TITLE
Add Maxim keymap for Fourier

### PR DESCRIPTION
**Pull request by nooges**

### CODE REVIEW

File: keyboards/fourier/keymaps/maxim/keymap.c [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
+#define _FN2 2
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+};
+
+#define _______ KC_TRNS
```
Comment (mechmerlin) [inline keyboards/fourier/keymaps/maxim/keymap.c]:
```suggestion

```

File: keyboards/fourier/keymaps/maxim/keymap.c [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+#define XXXXXXX KC_NO
```
Comment (mechmerlin) [inline keyboards/fourier/keymaps/maxim/keymap.c]:
```suggestion

```

File: keyboards/fourier/keymaps/maxim/config.h [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```
Comment (noroadsleft) [inline keyboards/fourier/keymaps/maxim/config.h]: `keyboards/fourier/config.h` includes this file, so it's redundant here.

```suggestion
```

File: keyboards/fourier/keymaps/maxim/config.h [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+/* Use I2C or Serial, not both */
+
+#define USE_SERIAL
+// #define USE_I2C
+
+#endif
```
Comment (noroadsleft) [inline keyboards/fourier/keymaps/maxim/config.h]:
```suggestion
```

**Comment by drashna** [comment]
Looks like there is a merge conflict here.

Also, any update on the changes listed here?

**Pull request merged**